historical_system_profiles/notification_service_interface.py

```diff
@@ -0,0 +1,92 @@
+import json
+
+from datetime import datetime, timezone
+from kafka import KafkaProducer
+
+from historical_system_profiles import config
+
+NOTIFICATION_BUNDLE = "insights"
```

Comment:
Current bundle in ci is `rhel` (previously it was `insights`)
https://ci.cloud.redhat.com/settings/notifications/rhel

Reply:
Is that value consistent across different envs (ci, qa, stage, prod)?

If not, we should make it to read from ENV.

Reply:
I have changed it to 'rhel' as per conversation with Thomas Heute

Reply:
> Is that value consistent across different envs (ci, qa, stage, prod)?

Yes, it's consistent across envs.

Reply:
...and it now also is an env variable